rest_framework/views.py

```diff
@@ -482,7 +482,7 @@ def raise_uncaught_exception(self, exc):
     # Note: Views are made CSRF exempt from within `as_view` as to prevent
     # accidental removal of this exemption in cases where `dispatch` needs to
     # be overridden.
-    def dispatch(self, request, *args, **kwargs):
+    def sync_dispatch(self, request, *args, **kwargs):
```

Comment:
We might want to keep this function named `dispatch` to prevent people's overrides from breaking.

Which means the only reasonable name for the other function is probably `adispatch`.

Reply:
Scratch that, overriding `dispatch` would still work for most people but would definitely lead to some unexpected behavior if they're trying to upgrade to async.

Don't think there's anything we can do about that.